cogs/audio.py

```diff
@@ -290,6 +290,9 @@ def _cache_max(self):
         setting_max = self.settings["MAX_CACHE"]
         return max([setting_max, self._cache_min()])  # enforcing hard limit
 
+    def _noppl_disconnect(self):
+        setting_nodis = self.settings["NOPPL_DIS"]
```

Comment:
`_noppl_disconnect ` is never called and even if it was, the setting is never put into the settings dict `self.settings` or into the `settings.json` file.